src/scratch/hvor_i_all_verden_del2/hvor_i_all_verden_2_nn.md

```diff
@@ -0,0 +1,342 @@
+*No kan me flytte kartet når helikopteret flyg rundt omkring.*
+
+## Sjekkliste {.check}
+
+Dette steget er eigentleg ganske lett. Det einaste me må passe på er at me skal
+flytte kartet __motsett__ veg av den helikopteret skal flyge. Til dømes må
+kartet flytte seg nedover viss me vil at helikopteret skal fly oppover.
+
+- [ ] Legg til denne koden på kartet:
+
+  ```blocks
+  når eg får meldinga [Nytt spel v]
+  vis
+  for alltid
+      gå til x: ((0) - (X)) y: ((0) - (Y))
+  slutt
+  ```
+
+## Test prosjektet {.flag}
+
+__Klikk på det grøne flagget.__
+
+- [ ] Ser det ut som om helikopteret flyttar på seg når du trykkar på
+  piltastane?
+
+- [ ] Sjå nøye etter. Ser du at helikopteret faktisk står på same stad heile
+  tida?
+
+- [ ] Kva skjer viss du flyr ut av kartet?
+
+## Utfordring {.challenge}
+
+Korleis kan du avgrense kor helikopteret flyr? Kan du leggje på kode slik at
+helikopteret ikkje kan flyge ut av kartet?
+
+
+# Steg 4: La staden følgje med {.activity}
+
+*Me må la reisemålet flytte seg med kartet.*
+
+## Sjekkliste {.check}
+
+- [ ] Sidan det er vanskeleg å sjå om reisemålet er med når det er usynleg, så
+  endrar me usynlegheita til stad-figuren. Til dømes kan du setje
+
+  ```blocks
+  set [gjennomsiktig v]-effekt til (50)
+  ```
+
+- [ ] For at det skal sjå ut som om staden ligg i ro på kartet må stad-figuren
+  flytte seg saman med kartet. Då treng me to variablar som heldt greie på kor
+  staden skal liggje i forhold til kartet. Lag to variablar som heiter `stadX`
+  og `stadY`. Begge desse skal berre gjelde for stad-figuren.
+
+- [ ] Byt ut
+
+  ```blocks
+  gå til x: (-98) y: (-120)
+  ```
+
+  med kode som set variablane i staden:
+
+  ```blocks
+  set [stadX v] til [-98]
+  set [stadY v] til [-120]
+  ```
+
+- [ ] Me kan lage ei heilt ny blokk som flyttar staden rundt på skjermen.
+
+  ```blocks
+  når eg får meldinga [Nytt spel v]
+  for alltid
+      gå til x: ((stadX) - (X)) y: ((stadY) - (Y))
+  slutt
+  ```
+
+  Prøv spelet. Flyttar den raude prikken seg rundt på kartet?
+
+- [ ] Du ser kanskje at den raude prikken ikkje ligg ved Barcelona lengre? Sidan
+  kartet er større stemmer ikkje dei gamle koordinatane.
+
+  Endre verdiane for `X` og `Y` til det ser ut som om reisemålet ligg omtrent
+  på riktig stad att.
+
+
+# Steg 5: Fleire stader {.activity}
+
+*No skal me leggje til fleire reisemål i spelet vårt.*
+
+## Sjekkliste {.check}
+
+Viss du berre ser på koden me har lagt på stad-figuren kan det synast som om det
+er mykje arbeidå leggje til eit nytt reisemål. Men om me ser nøyare på koden kan
+me sjå at ein ny stad kan bruke mykje av den same koden. Me må berre endre
+namnet (`Reis til`{.blockdata}) og koordinatane (`X`{.blockdata} og
+`Y`{.blockdata}).
+
+Dette er eit godt døme på når det kan løne seg å bruke funksjonar. I Scratch
+tilsvarar det å lage eigne klossar.
+
+- [ ] Klikk på `Flere klosser`{.blockmoreblocks} og lag ein kloss som ser slik
+  ut:
+
+  ![Bilete av ein ny kloss i Scratch](nykloss.png)
```

Comment:
Her tror jeg faktisk jeg må være så pirkete at du må legge til et nytt bilde. For på bildet står det  'sted' og ikke 'stad'